scenes/game_elements/characters/player/components/player.gd

```diff
@@ -13,6 +13,12 @@ enum Mode {
 	FIGHTING,
 }
 
+const REQUIRED_ANIMATION_FRAMES: Dictionary[StringName, int] = {
+	&"idle": 10,
+	&"walk": 6,
+	&"attack_01": 4,
+	&"attack_02": 4,
+}
```

Comment:
Why do we require these specific amount of frames in these animations?, I know that for the attack animation we use the frame number to decide if it can be cancellable, but why do we need the rest? 🤔

Reply:
> Why do we require these specific amount of frames in these animations?, I know that for the attack animation we use the frame number to decide if it can be cancellable, but why do we need the rest? 🤔

Because the frames of these animations are used in the timeline of the AnimationPlayer.

Reply:
Ohhh I see, okay then!, I think it's good to merge